test/e2e/config/multikueue/worker/kustomization.yaml

```
@@ -0,0 +1,16 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: kubeflow
+resources:
+- ../../../../../dep-crds/training-operator/base
+- namespace.yaml
+images:
+- name: kubeflow/training-operator
+  newTag: v1-9e52eb7
+secretGenerator:
+- name: training-operator-webhook-cert
+  options:
+    disableNameSuffixHash: true
+patchesStrategicMerge:
+- patch_crds.yaml
+- patch_deployment.yaml
```

```suggestion
apiVersion: kustomize.config.k8s.io/v1beta1
kind: Kustomization
namespace: kubeflow
resources:
  - ../../../../../dep-crds/training-operator/overlays/standalone
patchesStrategicMerge:
- patch_crds.yaml
- patch_deployment.yaml
```
The same situation here.